### PR TITLE
[rrd4j] Add console commands to check and clean RRD files

### PR DESCRIPTION
**Pull request by lolodomo**
Console commands added to detect and remove RRD files associated with items that no longer exist.

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Review by holgerfriedrich — APPROVED** [review]
A helpful addition! In the past I did this manually when upgrading old OH instances.
Code LGTM.

I gave it a try on my RPI test system, it worked as intended.

Maybe you could add some documentation? I was - until now - not aware that addons can easily add console commands.

File: bundles/org.openhab.persistence.rrd4j/src/main/java/org/openhab/persistence/rrd4j/internal/console/RRD4jCommandExtension.java [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+                    console.println("  - " + filename + ": file deletion failed!");
+                }
+            }
+        }
+        console.println(nb + " files " + (checkOnly ? "to delete." : "deleted."));
+    }
+
+    @Override
+    public List<String> getUsages() {
+        return Arrays.asList(buildCommandUsage(CMD_LIST, "list Round Robin Database files"),
```
Comment (wborn) [inline bundles/org.openhab.persistence.rrd4j/src/main/java/org/openhab/persistence/rrd4j/internal/console/RRD4jCommandExtension.java:142]:
```suggestion
        return List.of(buildCommandUsage(CMD_LIST, "list Round Robin Database files"),
```

Comment (lolodomo) [reply bundles/org.openhab.persistence.rrd4j/src/main/java/org/openhab/persistence/rrd4j/internal/console/RRD4jCommandExtension.java:142]: Changed

**Review by wborn — APPROVED** [review]
Thanks for the commands!

**Comment by wborn** [comment]
Yes you can use it by logging into the [Console](https://www.openhab.org/docs/administration/console.html).

The command has its own help:

```
openhab> openhab:rrd4j --help
Usage: openhab:rrd4j list - list Round Robin Database files
Usage: openhab:rrd4j check - check for RRD files without existing item
Usage: openhab:rrd4j clean [<itemName>] - delete RRD files without existing item
```

**Comment by B4dR4cc00n** [comment]
Thank you this is a very helpful command

**Pull request merged**